duckduckgo/macos-browser
# Add unit tests Github Actions workflow

Task/Issue URL: https://app.asana.com/0/1199230911884351/1202109118901706/f

**Description**:
The GitHub Actions workflow is set up to run on pushes to PR branches and `develop`.

**Steps to test this PR**:
1. Check "Checks" section on the bottom of this page.
1. Break a test and commit it to this PR branch. Wait for the workflow to run and observe failed test results. Check annotations appearing in the diff.
1. Fix broken test and recheck back on the build status. Notice failed tests annotations disappearing.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
